src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>5</number>
```

Comment:
Please edit this back to `0` so the first tab is the one selected initially. Whilst the code actually takes care of this anyhow this is an unneeded change...

:nerd_face: